Source/WTF/wtf/PlatformEnableCocoa.h

```diff
@@ -635,6 +635,10 @@
 #define ENABLE_MODEL_ELEMENT_ENVIRONMENT_MAP ENABLE_MODEL_PROCESS
 #endif
 
+#if !defined(ENABLE_MODEL_ELEMENT_IMMERSIVE) && PLATFORM(VISION)
+#define ENABLE_MODEL_ELEMENT_IMMERSIVE ENABLE_MODEL_PROCESS
```

Comment:
Do we want to enable this right away? `ENABLE_MODEL_PROCESS` is 1 on shipping visionOS